docs/src/development/transfer-dedicated.md

```diff
@@ -1,149 +1,135 @@
 ```bash
-ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u <user> -p<pass> -h localhost <dbname> | gzip' > database.gz
+   platform scp --project=<PROJECT_ID> --environment=<DEVELOPMENT_ENVIRONMENT> -r remote:<DIRECTORY_TO_SYNCHRONIZE> <LOCAL_DIRECTORY>
 ```
 
-That runs a `mysqldump` command on the server, compress it using `gzip`,
-and stream the output to a file named `database.gz` on your local computer.
+2. To copy the local directory to the remote Production mount:
 
-(If you'd prefer, `bzip2` and `xz` are also available.)
+```bash
+   platform scp --project=<PROJECT_ID> --environment=<PRODUCTION_ENVIRONMENT> -r <LOCAL_DIRECTORY> remote:<DIRECTORY_TO_SYNCHRONIZE>
+```
 
-## Synchronizing files from development to staging/production
+## Synchronize a database from Development to Staging/Production
 
-To transfer data into either the staging or production environments,
-you can either download it from your Platform.sh development environment to your local system first
-or transfer it directly between environments using SSH-based tools (such as scp, rsync).
+To synchronize a database into either the Staging or Production environments,
+you can export the database from your Development environment to your local system and from there to your Production environment.
 
-First, set up [SSH forwarding](./ssh/ssh-keys.md#forwarding-keys-by-default) by default for Platform.sh domains.
+{{< note theme="warning" >}}
 
-Then run `platform ssh` with the production branch checked out to connect to the default development environment.
-Files are the easier data to transfer, and can be done with `rsync`.
+Be aware that this is a destructive operation that overwrites data.
+[Backup your Production database first](#back-up-your-database).
 
-```bash
-rsync -avzP pub/static/ <USERNAME>@<CLUSTER_NAME>.ent.platform.sh:pub/static/
-```
+{{< /note >}}
 
-Replace `pub/static` with the path to your files on system, such as `web/sites/default/files/`.
-Note that rsync is very picky about trailing `/` characters.
-Consult the rsync documentation for more that can be done with that command.
+To synchronize your database:
 
-## Synchronizing the database from development to staging/production
+1. Export the Development database on your local computer:
 
-The database can be copied directly from the development environment to staging or production,
-but doing so requires noting the appropriate credentials first on both systems.
+   ```bash
+   platform db:dump --project=<PROJECT_ID> --environment=<DEVELOPMENT_ENVIRONMENT> --file=dump.sql
+   ```
 
-First, log in to the production environment over SSH:
+   For more options, see [database export](../add-services/_index.md#exporting-data).
 
-```bash
-ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh
-```
+2. Import the Development database dump file into the remote Staging/Production database:
+
+   ```bash
+   platform sql --project=<PROJECT_ID> --environment=<PRODUCTION_ENVIRONMENT> < dump.sql
```

Comment:
:clipboard: 
```suggestion
   platform sql --project=<PROJECT_ID> --environment=<ENVIRONMENT> < dump.sql
```
Since it's not just Production and including both would be too much